cmake/defaults/rv_options.cmake

```diff
@@ -11,6 +11,10 @@ OPTION(RV_SHOW_ALL_VARIABLES "Displays all build variables." ON)
 
 #
 # General build options
+
+OPTION(RV_USE_OCIO_OPENEXR "Uses OpenColorIO's own copy of OpenEXR." OFF)   # Mac & Windows do not work with OCIO's copy. Moreover, next release OCIO will remove OpenExr on their side.
```

Comment:
How about we remove RV_USE_OCIO_OPENEXR altogether since it is not currently used and won't be used in the future anyhow ?